src/interactive/repl.ts

```diff
@@ -376,6 +376,19 @@ function clearProgress() {
     }
 }
 
+function display(params: { kind: string, data: any }) {
+    if (params.kind === 'application/vnd.julia-vscode.trace') {
+        displayTrace(params)
+    } else {
+        plots.displayPlot(params)
+    }
+}
+
+function displayTrace(params: { kind: string, data: any }) {
+    results.clearStackTrace()
+    results.setStackFrameHighlight('', params.data)
```

Comment:
I want to have this like:
```julia
    results.setStackFrameHighlight(params.data.all, params.data.frames)
```
so that we can have messages for all the frames, but then it won't be clear how to handle multiple traces.

Maybe we can change the interface of `TRACE_MIME` so that it can take an array of traces ?

Reply:
Never mind, I think this will complicate the interface, and we can just attach message for each frame.